test/pylib/scylla_cluster.py

```diff
@@ -530,13 +530,14 @@ async def add_server(self) -> str:
         server = self.create_server(self.name, self._seeds())
         self.is_dirty = True
         try:
-            logging.info("Cluster %s adding server", server)
+            logging.info("Cluster %s adding server...", self)
```

Comment:
I think this this message is ambiguous - it looks like it prints which server is being added, but I understand correctly it actually prints the list of servers before adding the new one?

Reply:
It was supposed to print the cluster but there was a bug and it tried to print `server` (which gave an empty string at this point because `server` was not "installed" yet).

Now it's printing the cluster.

Printing the cluster meant printing the set of running servers in the cluster but I improved that as well.